class/defaults.yml

```
@@ -7,6 +7,11 @@ parameters:
       nodeSelector:
         node-role.kubernetes.io/infra: ''
     enableUserWorkload: false
+    upstreamRules:
+      networkPlugin: openshift-sdn
+      elasticsearchOperator: true
+      clusterSamplesOperator: true
+      openshiftLogging: true
```

```suggestion
      openshiftLogging: false
```

Logging is an optional component.

---

I'm aware of that, but this would be then a breaking change to our current config. I've preferred to do this in a second step together with changing also the network plugin to ovn-kubernetes by default.

---

Relates to openshift4-docs#118.

---

Cluster logging rules should be handled by openshift4-logging anyway, once we have proper infrastructure to allow components to inject rules, cf. https://github.com/projectsyn/component-rook-ceph/pull/3 for what I'm thinking of in terms of implementation.

---

OK, I see. Let us keep it for now. I agree with the idea to move the logging part to the logging component.